instructure/canvas-ios
# Fix Planner for manually enrolled observers

refs: MBL-14321
affects: parent
release note: Fixed Calendar for manually enrolled observers

Test plan:
* Log in as user from [KNO-403](https://instructure.atlassian.net/browse/KNO-403)
* April 10 and 11 should show planner items
* Use the Calendar filter to remove the only calendar
* April 10 and 11 events should be removed
